src/Effects.js

```diff
@@ -466,15 +466,18 @@ module.exports = class Effects {
             life: 5,
           });
         }
+        this.image = p5.createGraphics(75, 130);
+        this.image.scale(7);
```

Comment:
we set this scale a lot lower in the other emoji based foregrounds. have we tested this with music to see if the pineapples get to big?

Reply:
Good question!  I went with a higher number here to get more detail in the larger pineapples — the size of the drawn sprites shouldn't change.  The 7 is reduced back down by 1/7 on [line 479](https://github.com/code-dot-org/dance-party/pull/533/files#diff-c91a4859bf33bf9cb2bd5e367b7555a3R479).